src/util/futil.js

```diff
@@ -0,0 +1,13 @@
+import _ from 'lodash/fp'
+import F from 'futil'
+
+export let transformTreePostOrder = (next = F.traverse) =>
```

Comment:
This is [duplicated from `contexture-elasticsearch`](https://github.com/smartprocure/contexture-elasticsearch/blob/00e1a137d489d7d845c9b1881fefd0781cbdb101/src/utils/futil.js#L26-L31), but needs a some API bikeshedding to move to `futil`. An open question is whether to break the API in futil and accept always pre/post (changes current curry behavior), change the default to always be post, and/or make the fn be an object with `{pre, post}`.